### DISCUSSION TITLE
Pegs to show as columns in xsheet/timeline

### DISCUSSION THREAD

**Discussion by age3rcm** [opentoonz/opentoonz#6792] [Roadmap]
The pegs keyframes are even listed and able to edit on the function editor, and having the option to -select pegs only- will be a GREAT advantage for working with a cut-out puppet rig
Taking drawing columns as pegs already works,yes... but considering that you can select pegs individually would make this very handy considering the nature of a peg function.
it is a simple fix to show it on xsheet...

**Comment by RodneyBaker** [comment]
Adding roadmap label to this as @manongjohn has added pegs to timeline/xsheet in T2D.
Of importance, Opentoonz does not currently have folders so having pegs appear without ability to hide them or organize them will fall short.  As such it won't be enough to just have pegs appear in the timeline/xsheet. We'll also need to consider how to manage them (via folders, hiding, etc.)
